### PR TITLE
[READY] [TESTED] Adds a sheath to loadout.

### PR DESCRIPTION
**Pull request by Guidesu**
About The Pull Request
I coded a new sheath that is on loadout for 1 point, since it is technically a worse holster that can not be put on uniform and takes the belt or suit storage.

And reduced a few items prices to 0 that have or does not give any mechanical advantage

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by Guidesu** [comment]
UGGR
something horribly broke

File: code/game/objects/items/weapons/tools/simple_weapons.dm [hunk 1]
```diff
@@ -850,7 +850,7 @@
 /obj/item/tool/cheap/katana
 	name = "cheap katana"
 	desc = "A katana of acceptable quality, mass-produced by Lonestar. Easily their best-selling product from this lineup."
-	icon_state = "katana_old"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/simple_weapons.dm:853]: i like it using this sprite

File: code/modules/client/preference_setup/loadout/lists/armament.dm [hunk 2]
```diff
@@ -16,7 +16,7 @@
 /datum/gear/armament/ammo_h
 	display_name = "High Cap Rubber .35 Auto pistol mag"
 	path = /obj/item/ammo_magazine/highcap_pistol_35/rubber
-	cost = 2
+	cost = 1
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/armament.dm:19]: i am against this change

File: code/modules/client/preference_setup/loadout/lists/armament.dm [hunk 3]
```diff
@@ -61,7 +61,7 @@
 /datum/gear/armament/baton
 	display_name = "telescopic baton"
 	path = /obj/item/melee/telebaton
-	cost = 5
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/armament.dm:64]: I am against this change

**Pull request merged**